### PR TITLE
Setup code for chsh inequality

### PR DESCRIPTION
**Pull request by jkblc**
Setup code and start of backend connection. Should be compatible with previous pull request #29

### CODE REVIEW

File: chsh_inequality.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+
+# Plotting routines
+import matplotlib.pyplot as plt
+import matplotlib.ticker as tck
+
+# To run on hardware, select the backend with the fewest number of jobs in the queue
+service = QiskitRuntimeService(channel="ibm_quantum")
+backend = service.least_busy(operational=True, simulator=False, min_num_qubits=127)
```
Comment (laserlab) [inline chsh_inequality.py:20]: Could you set the minimum number of qubits to a more useful value for the Bell’s inequality problem?

**Review by laserlab — CHANGES_REQUESTED** [review]
Could you comment a little more what each line does using inline comments?

**Comment by laserlab** [comment]
Check PR #29 . you can add the setup code in a new file and then later the code will be put together in a new file.

**Comment by jkblc** [comment]
Yes, I'll pull the updated repository and add the setup code to quantumcode.py
I'll set the minimum qubits to 2 for now

**Pull request closed without merging**